src/appendix/2-how-to-use-gdb/README.md

```diff
@@ -0,0 +1,87 @@
+# How to use GDB
+
+Below are some useful GDB commands that can help us debug our programs. This assumes you have [flashed a program](../../05-led-roulette/flash-it.md) onto your microcontroller and attached to an OpenOCD session.
+
+## General Debugging
+
+> **NOTE:** Many of the commands you see below can be executed using a short form. For example, `continue` can simply be used as `c`, or `break $location` can be used as `b $location`. While you are learning GDB try to see how short you can get the commands to go before GDB doesn't recognize it!
```

Comment:
Not quite sure about this comment, users might end up with some confusing behaviour they don't really understand, I'd say getting the short forms of the command is not something for a person who's just learning GDB.

Reply:
You may be right, I have not removed it yet, but made a slight change in addition to getting rid of all the shortcuts in the main section. If you still think it could lead to any confusion, let me know and we can take it out. Thanks.